README.md

```diff
@@ -68,3 +73,48 @@ You can find the greentea-client example application [here](./example/main.cpp).
     cmake --build cmake_build
 
 This generates an executable `./cmake_build/example/greentea-client-example`.
+
+# Adding greentea-client to a project
+
+## Build support
+
+greentea-client supports the CMake build system. To add greentea-client to a project, call
+[`add_subdirectory`](https://cmake.org/cmake/help/latest/command/add_subdirectory.html)
+pointing to the path to the client. Then link `greentea-client` to a CMake target that requires it with
+[`target_link_libraries`](https://cmake.org/cmake/help/latest/command/target_link_libraries.html).
```

Comment:
Having examples of both `add_subdirectory(greentea-client)` and `target_link_libraries(my_app greentea-client)` would help with clarity.

Reply:
Added